docs/PfSense.md

```diff
@@ -0,0 +1,35 @@
+In order to boot PfSense via PXE,
+you need to do the following:
+
+1. Download ISO from https://nyifiles.pfsense.org/mirror/downloads/pfSense-CE-2.3.4-RELEASE-amd64.iso.gz
+   
+   You should also check the SHA256 hash:
+   `sha256 -c 610b2a8c696e1d1854845d553ce8540debaab9d78ed6c15ce83872eaeac9d05f pfSense-CE-2.3.4-RELEASE-amd64.iso.gz`
```

Comment:
I believe this is redundant. Please choose between providing the hash in command to execute or asking people to check it using the site. IMHO, I'm not a huge fan of putting such "harcoded" values in documentation (it's doing redundancy with the image owner's site).

Reply:
I have removed the info about website. SHA256 is hardcoded, because the command uses it.